crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -112,7 +112,8 @@ impl DWalletMPCManager {
                 NetworkDkg::init(epoch_store.clone())?,
             )
         } else {
-            // Todo (#380): Load the network DKG outputs
+            // Todo (#382): Store the real value of the decryption key shares
+            let _ = epoch_store.get_encrypted_decryption_key_shares();
```

Comment:
print the error at least.

Reply:
There is no error. Simply the DKG shouldn't be triggered, but we already changed this logic